src/main/java/frc/robot/targeting/NodePose.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.targeting;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+import java.util.Objects;
+
+import frc.robot.targeting.NodePoses.NodeID;
+
+/**
+ * 
+ *
+ * @author aheitkamp
+ */
+public class NodePose {
+    private final Pose2d poseToTag;
+    private final NodeID id;
+
+    public NodePose(double horizontalOffsetInches, double verticalOffsetInches, double Radians, NodeID ID) {
+        poseToTag = new Pose2d(horizontalOffsetInches, verticalOffsetInches, new Rotation2d(Radians));
+        id = ID;
+    }
+
+    public NodeID getNodeID() { return id; }
```

Comment:
please reformat to sun java coding standards-- no single-line functions.  use { on end of line. VS code should have an autoformatter that will apply the right standards